app/views/users/show.html.erb

```diff
@@ -5,3 +5,19 @@
     <br>
     <%= @user.email %>
     
+    <% if @user == current_user %>
+        <table>
+            <% Item.where(user: @user).find_each do |item| %>
```

Comment:
Not sure what the Rails convention is here, but I have seen the slightly more readable:
```ruby
# in controller
def show
  @user = User.find_by(params[:id])
  @items = Item.where(user: @user)
end
```
```erb
<!-- in view -->
<% @items.each do |item| %>
  <!-- do something -->
<% end %>
```